qir/stdlib/src/stdlib.c

```diff
@@ -2,7 +2,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-
+/*
```

Comment:
It looks like some of these changes may have accidentally made it into #40

Reply:
Will have a look.

Reply:
Removed from #40.